Dockerfile

```diff
@@ -1,22 +1,8 @@
 FROM ubuntu:22.04 AS skiplang-base
 
 ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get update && \
-    apt-get install -q -y wget gnupg && \
-    wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
-    echo "deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-15 main" >> /etc/apt/sources.list.d/llvm.list && \
-    echo "deb-src http://apt.llvm.org/jammy/ llvm-toolchain-jammy-15 main" >> /etc/apt/sources.list.d/llvm.list && \
-    apt-get update && \
-    apt-get install -q -y automake clang-15 clang-format-15 curl file gawk gcc git jq lld-15 llvm-15 make parallel unzip zip
-
-RUN update-alternatives --install /usr/bin/clang clang /usr/bin/clang-15 100 && \
-    update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-15 100 && \
-    update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-15 100 && \
-    update-alternatives --install /usr/bin/llc llc /usr/bin/llc-15 100 && \
-    update-alternatives --install /usr/bin/llvm-ar llvm-ar /usr/bin/llvm-ar-15 100 && \
-    update-alternatives --install /usr/bin/llvm-config llvm-config /usr/bin/llvm-config-15 100 && \
-    update-alternatives --install /usr/bin/llvm-link llvm-link /usr/bin/llvm-link-15 100 && \
-    update-alternatives --install /usr/bin/wasm-ld wasm-ld /usr/bin/wasm-ld-15 100
+COPY ./bin/apt-install.sh /tmp/
+RUN /tmp/apt-install.sh llvm
```

Comment:
Just curious, why copy instead of running it from the original location?

Reply:
Like that?
```
RUN --mount=type=bind,source=./bin/apt-install.sh,target=/tmp/apt-install.sh /tmp/apt-install.sh llvm
```

Reply:
Sorry, I didn't expect it to be so involved. I was thinking `RUN ./bin/apt-install.sh`... :-)